tests/suites/config.js

```diff
@@ -1,39 +1,40 @@
-module.exports = [{
+module.exports = [
+{
 	deviceType: process.env.DEVICE_TYPE,
-	suite: `${__dirname}/../suites/os`,
+	suite: `${__dirname}/../suites/hup`,
```

Comment:
@klutchell mayeb add to the commit what's the problem with the current suite ordering?

Reply:
Sure, I'll update the commit message. Nothing wrong with the ordering but I consider the unmanaged tests the highest priority and like to see them run first. Especially if we decide to failfast in the future.